modules/ztp-recommended-cluster-kernel-config.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-vdu-validating-cluster-tuning.adoc
+
+:_module-type: REFERENCE
+[id="ztp-recommended-cluster-kernel-config_{context}"]
+= Recommended cluster kernel configuration
+
+Always use the latest supported real-time kernel version in your cluster. You should also ensure that the following configurations are applied in the cluster:
```

Comment:
Nit: See the [ISG guidance](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#should) on _should_.